test/integration/concrete/lockup/withdraw-max-and-transfer/withdrawMaxAndTransfer.tree

```diff
@@ -8,9 +8,9 @@ withdrawMaxAndTransfer.t.sol
       ├── when the caller is not the current recipient
       │  └── it should revert
       └── when the caller is the current recipient
-         ├── when the NFT has been burned
+         ├── given the NFT has been burned
          │  └── it should revert
-         └── when the NFT has not been burned
+         └── given the NFT has not been burned
             ├── when the withdrawable amount is zero
```

Comment:
Should be "given when the withdrawable amount is zero" since this is a background property.

Reply:
Right. Changed the modifier name as well: https://github.com/sablier-labs/v2-core/pull/657